README.md

```diff
@@ -1,24 +1,43 @@
 # Go Rules
-This repo provides Golang build rules for the [Please](https://please.build) build system. 
 
-# Basic usage
+This repo provides [Go](https://go.dev/) build rules for the [Please](https://please.build) build system.
+
+Go is especially suited to writing command line tools and utilities. Binaries can be run with `plz run`, or used
+as tools for [custom rules](https://please.build/codelabs/genrule/#0).
+
+# Installation
```

Comment:
Can we just recommend using `plz init plugin go` here? (and maybe leave the manual process as an appendix)